tests/cxx/relative_include_of_export_3.cc

```diff
@@ -0,0 +1,21 @@
+//===--- relative_include_of_export-d1.h - test input file for iwyu -------===//
```

Comment:
For now I've just opted for really long names.  We'll see what you think.

Reply:
Generally, I think they're too long: see guidelines. But I also know it's hard to find short, meaningful names, so I'm not going to veto anything.

I wonder if it would help/be possible to merge all these into a single testcase for the path mapping features of IWYU? Maybe there already are testcases related to this that could be extended?